crates/lib/src/executable.rs

```diff
@@ -119,9 +121,10 @@ impl<'executable> Executable<'executable, '_> {
             params: Parameters::new(),
             compile_with_quilc: true,
             compiler_options: CompilerOpts::default(),
-            client: None,
             qpu: None,
             qvm: None,
+            qcs_client: None,
+            quilc_client: None,
```

Comment:
If we use the existence of the client to determine whether or not to compile with quilc, should we remove the `compile_with_quilc` property from the `Executable` struct? Also, compiling with quilc was the default, so should we instantiate a `quilc_client` by default?